notebooks/tutorial/7 - Getting data from the ADX.ipynb

```diff
@@ -619,10 +619,10 @@
     "There are many many ways to do this so we won't list them all here. We generally sync one month at a time with the AWS CLI like this:\n",
     "\n",
     "```\n",
-    "aws s3 sync s3://my-bucket/csds/2022/04/ /path/to/working/dir/csds/2022/04\n",
+    "aws s3 sync s3://my-bucket/csds/2022/04/ /PATH/TO/CSDS/csds/2022/04\n",
     "```\n",
     "\n",
-    "It is **CRITICALLY IMPORTANT** to maintain the path structure. In the example above, you could change the `/path/to/working/dir` but the `csds/2022/04` is not something that you may change. The reader will not work because the folder structure is part of the key used to read each file."
+    "It is **CRITICALLY IMPORTANT** to maintain the path structure. In the example above, you could change the `/PATH/TO/CSDS` but the `csds/2022/04` is not something that you may change. The reader will not work because the folder structure is part of the key used to read each file."
```

Comment:
```suggestion
    "It is **CRITICALLY IMPORTANT** to maintain the path structure. In the example above, you could change the `/PATH/TO/ROOT` but the `csds/2022/04` is not something that you may change. The reader will not work because the folder structure is part of the key used to read each file."
```